pkg/webui/console/views/gateway-overview/gateway-overview.js

```diff
@@ -120,6 +174,24 @@ export default class GatewayOverview extends React.Component {
             key: sharedMessages.frequencyPlan,
             value: frequency_plan_id ? <Tag content={frequency_plan_id} /> : undefined,
           },
+          {
+            key: m.globalConf,
+            value: this.state.globalConf ? (
+              <ModalButton
+                type="button"
+                secondary
+                message={m.showGlobalConf}
+                modalData={{
+                  danger: false,
+                  buttonMessage: m.downloadGlobalConf,
+                  children: global_conf_code,
+                }}
+                onApprove={this.handleDownload}
+              />
+            ) : (
+              undefined
+            ),
+          },
```

Comment:
1.  The whole row should be hidden when there is no config (not just the value)
2.  After understanding the use case for the conf, let's keep this simple: I think it's better to show two buttons, one for copying, one for downloading the conf. I understand that it's not really necessary to inspect the value directly inside the console, but correct me if you think I'm wrong here.
3.  After all, I don't think the `<ModalButton />` is the right choice for this. It's quite misappropriated in this implementation.